crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/entry_point.rs

```diff
@@ -185,7 +187,6 @@ pub fn execute_call_entry_point(
                     context,
                 )
             } else {
-                println!("Executing native entry point");
```

Comment:
nit: pls remove this in target branch (`2714-cairo-native`)

Reply:
Don't want to rebase the whole stack 😅